src/backend/env/mod.rs

```diff
@@ -274,6 +274,42 @@ pub enum Destination {
 }
 
 impl State {
+    pub fn delay_weekly_chores(&mut self, caller: Principal) -> bool {
```

Comment:
Should we delay the chores by one week only? Why not one day or any other delay?

Reply:
My assumption was that we would only ever use this mechanism if we determine some defect or vulnerability in Taggr's smart-contract. This would always require an upgrade. Since we cannot predict how long a fix would take, one week sounds like a safe bet.